app/routes/index/chapter/question.js

```diff
@@ -52,20 +52,51 @@ export default Ember.Route.extend({
       // Input fields use this to update the tag it's working on
       window.console.log("Updating tag locally goes here...");
 
-      tag.set('answer', option.get('value'));
+      tag.set('answer', [option.get('value')]);
+    },
+    clearTag(tag, option) {
```

Comment:
Is this necessary any more? Answers are removed via the saveTag() method now. Please remove if it is not needed.